### PR TITLE
add Computer Basic Logic as described in the requirement doc

### PR DESCRIPTION
**Pull request by cyberj19**
check out the console to see computer's cards and selections judgment.

### CODE REVIEW

File: objects/computer.js [hunk 1]
```diff
@@ -9,29 +9,47 @@ Computer.prototype = Object.create(Deck.prototype);
...
+    console.log('COMPUTER TURN');
     !heap.takiMode && Sound.tik();
     this._turn = 1;
     this.endFunction = endFunction;
-    this.cards.forEach(function (card) {
-        if (heap.isCardEligible(card) && card.type !== 'COLOR')  // temporary role not to choose color card
-        {
-            availableCards.push(index);
+    // TODO: change to var instead of let
+    let maxScore = -1;
```
Comment (cuzi) [inline objects/computer.js]: change to var instead of let

File: objects/computer.js [hunk 2]
```diff
@@ -41,4 +59,13 @@ Computer.prototype.endTurn = function (endCard) {
...
-};
\ No newline at end of file
+};
+
+var cardScore = function(card, heap) { // this function runs for eligible cards only
+    if (card.type === "COLOR") return 6;
+    if (card.type === "STOP") return 5;
+    if (card.type === "+") return 4;
+    if (card.type === "TAKI") return 3;
+    if (card.color === heap.color) return 2;
```
Comment (cuzi) [inline objects/computer.js:68]: you don't have property "color" in the heap

File: objects/computer.js [hunk 3]
```diff
@@ -9,29 +9,47 @@ Computer.prototype = Object.create(Deck.prototype);
...
             this.endTurn();
         }
         else this.pullCard();
+        return;
+    }
+    let priorityCard = this.cards[priorityIndex];
+
+    if (priorityCard.type === 'COLOR') {
+        let colors = cardsColors();
+        priorityCard.color = colors[Math.floor(Math.random() * colors.length)];
```
Comment (cuzi) [inline objects/computer.js]: Choose color by random it is really unnecessary we need to use the color card only if there is a need to change color and to change it to a color that we got in our pack, else it is waste of that card..

**Comment by cyberj19** [comment]
fix

**Pull request closed without merging**